charts/br-sta/values.yaml

```
@@ -0,0 +1,727 @@
+  observability:
+    # -- Enable telemetry export (ENABLE_TELEMETRY). Unset → chart default "false".
+    enabled: false
+    # -- OTLP collector endpoint (OTEL_EXPORTER_OTLP_ENDPOINT). Unset → chart default "".
+    otlpEndpoint: ""
+    # -- Deployment environment tag (OTEL_RESOURCE_DEPLOYMENT_ENVIRONMENT). Unset → "production".
+    deploymentEnvironment: "production"
+  # -- Env-wide multi-tenant infra (tenant-manager URL + its Redis), consumed by
+  # lerian-common.multiTenant.env. Only used when MT is enabled (see
+  # br-sta.multiTenant.enabled). A component configmap.<KEY> still overrides.
+  multiTenant: {}
+    # url: ""             # MULTI_TENANT_URL (required when MT enabled)
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)